docs/reference/api/workspaces.md

```diff
@@ -23,25 +23,25 @@ of the template will be used.
 
 ```json
 {
-	"automatic_updates": "always",
-	"autostart_schedule": "string",
-	"name": "string",
-	"rich_parameter_values": [
-		{
-			"name": "string",
-			"value": "string"
-		}
-	],
-	"template_id": "c6d67e98-83ea-49f0-8812-e4abae2b68bc",
-	"template_version_id": "0ba39c92-1f1b-4c32-aa3e-9925d7713eb1",
-	"ttl_ms": 0
+  "automatic_updates": "always",
```

Comment:
Why did these indentations change from tab to space? I saw other markdown docs that retained tab indentation for JSON code blocks and tabs is what we use mostly across the code base.

Reply:
reverted.